dracut/modules.d/99kiwi-dump-reboot/kiwi-dump-reboot-system.sh

```diff
@@ -7,6 +7,10 @@ type run_dialog >/dev/null 2>&1 || . /lib/kiwi-dialog-lib.sh
 #======================================
 # Functions
 #--------------------------------------
+function get_kernel_version_majorminor {
+	printf "%d%02d" "$(uname -r | cut -f1 -d.)" "$(uname -r | cut -f2 -d.)"
```

Comment:
I guess having two digits for the minor version is enough, let's hope something like kernel `v5.99` or `v5.125` won't ever exist or, at least, is not about to happen at mid term :)

Reply:
You are right this code has some potential to return numbers which doesn't serve the purpose for a comparison as number. Maybe it should not be done. How about this information:

```
 -a (--kexec-syscall-auto)
              Try the new KEXEC_FILE_LOAD syscall first and  when  it  is  not
              supported  or  the kernel does not understand the supplied image
              fall back to the old KEXEC_LOAD interface.

              There is no one single interface that always works.

              KEXEC_FILE_LOAD is required  on  systems  that  use  locked-down
              secure  boot  to verify the kernel signature.  KEXEC_LOAD may be
              also disabled in the kernel configuration.

              KEXEC_LOAD is required for some  kernel  image  formats  and  on
              architectures that do not implement KEXEC_FILE_LOAD.

```

@Conan-Kudo @davidcassany Thoughts ?

Reply:
This is probably the best we could do...

Reply:
hmm, Linus did never do more than 2 digit minor version number so far. 

i fear this -a option was only added in later kexec binaries perhaps and might not be available in older ones? Otherwise it would be nice to have that.

Reply:
sounds good to me

@msmeissn would you like to update the change in a way suggested by David or would you rather like me to take the current code and move it into a branch from where I have permissions to provide a new PR ? Either way is fine for me and I don't want to stress you with even more effort on this one. Just let us know what you prefer ?

Thanks

Reply:
Before entering vacation I'd like to open the alternative way suggested by @davidcassany. I hope you don't mind that I created #2155 as another possible solution. Let's decide what we prefer.

Thanks